modApiServer/test/org/aion/api/server/TxRecptLgTest.java

```diff
@@ -0,0 +1,162 @@
+import org.aion.zero.types.AionTransaction;
+import org.aion.zero.types.AionTxReceipt;
+import org.junit.Assert;
+import org.junit.Test;
+import org.aion.base.type.Address;
+
+
+
+public class TxRecptLgTest {
+
+/*
+pragma solidity ^0.4.0;
+
+contract A{
+    event AE(uint128 value);
+    event AEA(uint128 value);
+    event AEB(uint128 value);
+
+    function AA(address addr){
+        AE(1);
+        AEA(2);
+        //B(addr).BB(addr, 1, 256, 128)
+        addr.call.value(0)(bytes4(sha3("BB(address,uint128,bytes32,bytes32)")), addr, 1, 256, 128);
+        AEB(3);
+    }
+}
+
+contract B{
+    event BE(address indexed _recipient, uint128 indexed _amount, bytes32 indexed _foreignNetworkId, bytes32 _foreignData);
+
+    function BB(address _recipient, uint128 _amount, bytes32 _foreignNetworkId, bytes32 _foreignData){
+        BE(_recipient, _amount, _foreignNetworkId, _foreignData);
+    }
+}
+*/
+
+    @Test
+    public void TestTxRecptLg() throws InterruptedException {
+        System.out.println("run TestTxRecptLg.");
+
+        String contractA = "0x605060405234156100105760006000fd5b610015565b610210806100246000396000f30060506040526000356c01000000000000000000000000900463ffffffff1680639dacb3a3146100335761002d565b60006000fd5b341561003f5760006000fd5b61005d6004808080601001359035909160200190919290505061005f565b005b7fcd6e704bef76e0fe8cfd711b498592ac8e681ff2aaace78726cbb1caea41249660016040518082815260100191505060405180910390a17f5e14984659fb7f7b895bdd08828f5de80edca21fa3f8d58bac20f83fc95a246e60026040518082815260100191505060405180910390a18181600060405180806f424228616464726573732c75696e743181526010016f32382c627974657333322c627974657381526010016f333229000000000000000000000000008152601001506023019050604051809103902090506c010000000000000000000000009004908585600161010060806040518763ffffffff166c01000000000000000000000000028152600401808686825281601001526020018460ff1681526010018361ffff1681526010018260ff168152601001955050505050506000604051808303818589895af1945050505050507fd8d8edb52a0e15005d291f5cb67640400ee21833034b9f09c4afad215fcffffe60036040518082815260100191505060405180910390a15b50505600a165627a7a72305820ac4d58f9a803c3144f57ca227ff9dbd6e121d52a10c070a22dc447ae219b97660029";
+        String contractB = "0x605060405234156100105760006000fd5b610015565b61011f806100246000396000f30060506040526000356c01000000000000000000000000900463ffffffff1680637fe7a5a014603157602b565b60006000fd5b3415603c5760006000fd5b608f60048080806010013590359091602001909192908035906010019091908080601001359035906000191690909160200190919290808060100135903590600019169090916020019091929050506091565b005b83839060001916908660008a8a7ff2433e32c57456caf7380aef3887a965ac2021deb7bf0989ee41a2cf4387b2f88989604051808383906000191690906000191690825281601001526020019250505060405180910390a45b505050505050505600a165627a7a723058209ec9a5d885689d4f729e93521792f26ebb06e8bc0d4b245489b602ddb289a1000029";
+
+        StandaloneBlockchain.Bundle bundle = (new StandaloneBlockchain.Builder())
```

Comment:
don't need (  ) around the new

Reply:
Thanks for the comments! The test takes 2-3 seconds to run.